main.md

```diff
@@ -180,7 +180,7 @@ locations:
 actions:
 :   An array of strings representing the kinds of actions to be taken at the resource. The values of the strings are determined by the API being protected.
 
-data:
+datatype:
```

Comment:
what's the difference between this and the "type" field?

Reply:
The "type" is the type of the overall object request, "datatype" is the kinds of data being asked for. So in OIDC this would be "profile", "email", "address", etc, those are all "datatype" style elements.

Reply:
So this is intended to be used in cases where the client requests access to certain claims, only? My impression so far was that this is a generic container to pass authz data.

Reply:
Claims are just one (identity-specific) kind of "datatype" that the client could ask for. Imagine a photo storage system, you'd have "image" data and "location" data and "timestamp" data all on the same resources.

Reply:
Would you mind to add an example? I assume I won't be the only one mixing requests for data elements at the RS and parameters to the authz process.